src/carplay.cs

```diff
@@ -164,6 +164,97 @@ public enum CPTextButtonStyle : long
 		Confirm,
 	}
 
+	[NoWatch, NoTV, NoMac, iOS (15,0)]
+	[Native]
+	public enum CPAssistantCellPosition : long
+	{
+		Top = 0,
+		Bottom,
+	}
+
+	[NoWatch, NoTV, NoMac, iOS (15,0)]
+	[Native]
+	public enum CPAssistantCellVisibility : long
+	{
+		Off = 0,
+		WhileLimitedUIActive,
+		Always,
+	}
+
+
+	[NoWatch, NoTV, NoMac, iOS (15,0)]
+	public enum CPJunctionType : byte
+	{
+		Intersection = 0,
+		Roundabout = 1,
+	}
+
+	[NoWatch, NoTV, NoMac, iOS (15,0)]
+	public enum CPTrafficSide : byte
+	{
+		Right = 0,
+		Left = 1,
+	}
+
+	[NoWatch, NoTV, NoMac, iOS (15,0)]
+	public enum CPManeuverType : byte
```

Comment:
So the "uint8_t" type should be "byte" in our bindings?

Reply:
yes, `u` for unsigned and `8` for 8-bits -> `System.Byte`